app/models/miq_report_result.rb

```diff
@@ -35,6 +35,9 @@ class MiqReportResult < ApplicationRecord
     end
   end
 
+  before_save  { @_extra_groupings = report.extras.delete(:grouping) if report.kind_of?(MiqReport) }
```

Comment:
I feel like this needs a short comment about why we're doing this and a link to the BZ.